modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
+----
++--------------------------------------------+
+| name     | currentStatus | requestedStatus |
++--------------------------------------------+
+| "movies" | "online"      | "online"        |
+| "neo4j"  | "online"      | "online"        |
++--------------------------------------------+
+----
+
+[NOTE]
+====
+Note that for failed databases, the `currentStatus` and `requestedStatus` are different.
+This often implies an error, but **that is not always the case**.
+For example, a database may take a while to transition from `offline` to `online` due to a performing recovery.
+Or, during normal operation, a database's `currentStatus` may be transiently different from its `requestedStatus` due to a necessary automatic process, such as one Neo4j instance copying store files from another.
+The possible statuses are `initial`, `online`, `offline`, `store copying` and `unknown`.
+====
+
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-create]]
+== Create databases
+
+Databases can be created using `CREATE DATABASE`.
+
+[NOTE]
+====
+Having dots (`.`) in the database names is not recommended.
+This is due to the difficulty of determining if a dot is part of the database name or a delimiter for a database alias in a composite database.
+====
+
+.Query
+[source, cypher]
+----
+CREATE DATABASE customers
+----
+
+When a database has been created, it will show up in the listing provided by the command `SHOW DATABASES`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES YIELD name
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+----
++-------------+
+| name        |
++-------------+
+| "customers" |
+| "movies"    |
+| "neo4j"     |
+| "system"    |
++-------------+
+----
+
+[[manage-databases-existing]]
+=== Use `IF EXISTS` or `OR REPLACE` when creating databases
+
+Appending `IF NOT EXISTS` to the `CREATE DATABASE` command ensures that no error is returned and nothing happens should the database already exist.
+
+.Query
+[source, cypher]
+----
+CREATE DATABASE customers IF NOT EXISTS
+----
+
+Adding `OR REPLACE` to the command results in any existing database being deleted and a new one being created.
+
+.Query
+[source, cypher]
+----
+CREATE OR REPLACE DATABASE customers
+----
+
+This is equivalent to running `DROP DATABASE customers IF EXISTS` followed by `CREATE DATABASE customers`.
+
+These commands are optionally idempotent, with the default behavior to fail with an error if the database already exists.
+
+These behavior flags apply to both standard and composite databases (e.g. a composite database may replace a standard one or another composite).
+
+[NOTE]
+====
+The `IF NOT EXISTS` and `OR REPLACE` parts of these commands cannot be used together.
+====
+
+[[manage-databases-create-database-options]]
+=== `CREATE DATABASE` options
+
+The `CREATE DATABASE` command can have a map of options, e.g. `OPTIONS {key: 'value'}`.
+
+[NOTE]
+====
+There are no available `OPTIONS` values for composite databases.
+====
```

Comment:
do you need this note here? it feels like it belongs to the composite database section (where I think it was as well?)